frb_example/pure_dart/rust/src/api.rs

```diff
@@ -1695,3 +1695,17 @@ pub fn test_tuple_2(value: Vec<(String, i32)>) {
 pub fn sync_return_mirror() -> SyncReturn<ApplicationSettings> {
     SyncReturn(external_lib::get_app_settings())
 }
+
+macro_rules! generate_struct {
```

Comment:
nit: would be great to test more, e.g. the marker macros you mentioned

(surely not now, but when we are sure this way does work! just want to add this sentence now in case I forget in the future)

Reply:
you mean the markers (like mirror) inside macro generated Code? I can add that.

Reply:
added